.gitmodules

```diff
@@ -14,3 +14,7 @@
 	path = vendor/github.com/shazow/rateio
 	url = https://github.com/shazow/rateio
 	branch = master
+[submodule "vendor/github.com/dustin/go-humanize"]
```

Comment:
Try doing `git submodule add https://github.com/dustin/go-humanize vendor/github.com/dustin/go-humanize`, I think it needs to pin the commit somewhere too.